dynatrace-aws-logs.sh

```diff
@@ -214,11 +227,17 @@ EOF
   fi
 
   TARGET_URL=$(echo "$TARGET_URL" | sed 's:/*$::')
+
+  if [[ "$USE_EXISTING_ACTIVE_GATE" == "true" ]]; then
+    check_activegate_state
+  fi
```

Comment:
I'd move it 239 after Marysia's check_log_ingest_url - it seems to me to make sense to verify the TARGET_URL format first  & there's the same if

Reply:
Verification of URL pattern is in line 224 - above.

I think current order is correct as it starts from basic check into final log ingest check (next check is depending on conditions checked in previous one):
- check ActiveGte status
- check token permission
- try to ingest test log line

Reply:
Missed it, thanks.